### ISSUE TITLE
Keywords get hidden in the comments

### ISSUE THREAD

**Issue by GTValentine** [hhatto/autopep8#462]
I ran autopep8 on [pycodestyle.py](https://github.com/PyCQA/pycodestyle/blob/bd17ca651a57f8cdd278a8fac84d6a09a178a3eb/pycodestyle.py) and it broke the code. The diff is:

```
@@ -1119,9 +1119,9 @@ def compound_statements(logical_line):
     counts = {char: 0 for char in '{}[]()'}
     while -1 < found < last_char:
         update_counts(line[prev_found:found], counts)
-        if ((counts['{'] <= counts['}'] and   # {'a': 1} (dict)
-             counts['['] <= counts[']'] and   # [1:2] (slice)
-             counts['('] <= counts[')'])):    # (annotation)
+        if ((counts['{'] <= counts['}']   # {'a': 1} (dict) and
+             counts['['] <= counts[']']   # [1:2] (slice)
+             and counts['('] <= counts[')'])):    # (annotation)
```
My .pep8 file:
```
[pep8]
indent-size = 4
max-line-length = 120
ignore = E3
```

You can see that it put `and` in the comment and broke the code.

**Comment by hhatto** [comment]
@GTValentine 
Thanks for your bug reporting.

This problem fix in #442 .
It is scheduled to be released in version 1.4.4 or 1.5.

**Comment by hhatto** [comment]
fix in version 1.5
